app/src/main/java/com/training/listviewadapter/MainActivity.java

```diff
@@ -3,36 +3,101 @@
 import android.app.ListActivity;
 import android.os.Bundle;
 import android.view.View;
-import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.EditText;
 import android.widget.ListView;
-import android.widget.TextView;
+import android.widget.Toast;
+
+import java.util.ArrayList;
+import java.util.List;
 
 public class MainActivity extends ListActivity {
 
-    private TextView selectedItemTextView;
+    private EditText name_editText;
+    private EditText age_editText;
+    private EditText gender_editText;
+    private Button addButton;
+    private Button resetButton;
+    private Button removeButton;
 
-    private static final String[] items = {"lorem", "ipsum", "dolor",
-            "sit", "amet",
-            "consectetuer", "adipiscing", "elit", "morbi", "vel",
-            "ligula", "vitae", "arcu", "aliquet", "mollis",
-            "etiam", "vel", "erat", "placerat", "ante",
-            "porttitor", "sodales", "pellentesque", "augue", "purus"};
+    private List<Person> personList = new ArrayList<>();
+    private PersonAdapter adapter;
 
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
-        selectedItemTextView = findViewById(R.id.section_text_view);
+        name_editText = findViewById(R.id.name_Edit_Text);
+        age_editText = findViewById(R.id.age_Edit_Text);
+        gender_editText = findViewById(R.id.gender_Edit_Text);
+        addButton = findViewById(R.id.add_button);
+        resetButton = findViewById(R.id.reset_button);
+        removeButton = findViewById(R.id.remove_button);
+
+        adapter = new PersonAdapter(this, personList);
+        setListAdapter(adapter);
+
+        addButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                onAddClicked();
+            }
+        });
+
+        resetButton.setOnClickListener(new View.OnClickListener() {
```

Comment:
I'd recommend that you implement View.OnclickListener so that you can have a single onClick() method which will then be applied to both buttons